analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -348,11 +348,17 @@ data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val authors = json["authors"]?.textValue()
+                ?.split(',')
+                ?.map {
```

Comment:
The lambda of the `map()` call could go into a single line and mapTo() could be used like `?.mapTo(sortedSetOf()) { it.trim() }`.

If there is need to filter empty strings, you could do:
`?.mapNotNullTo(sortedSetOf()) { it.trim().takeIf { it.isNotEmpty() } }`

Finally, consider eliminating the question marks and the `?:` at the end by using `textValueOrEmpty()`.